msal/src/telemetry/java/com/microsoft/identity/client/ApiEvent.java

```diff
@@ -44,7 +44,9 @@ private ApiEvent(Builder builder) {
         if (null != builder.mRequestId) {
             setProperty(EventProperty.REQUEST_ID, builder.mRequestId);
         }
-        setProperty(EventProperty.AUTHORITY_NAME, HttpEvent.sanitizeUrlForTelemetry(builder.mAuthority));
+        if (!MsalUtils.isEmpty(builder.mAuthority)) {
```

Comment:
why builder.mAuthority could be empty? if this is the API event, you should always have the authority.